### ISSUE TITLE
Will a version of this add-on compatible with NVDA 2025.x be forthcoming?

### ISSUE THREAD

**Issue by britechguy** [CoBC/easyTableNavigator-1#19]
The title pretty much covers it.  A great many people use this add-on.  It's better that they not have to use the override incompatibility feature to keep using it.

**Comment by josephsl** [comment]
Hi,

I would like to second the issue submitter - we have folks lined up to offer pull requests for this add-on, and I think we should not proceed with PR ideas until the add-on declares 2025.1 support. If it cannot be done due to NVDA Core changes, let us (add-ons community members) know and we can help.

Thanks.
